### PR TITLE
#167377502 User can comment on articles

### PR DESCRIPTION
**Pull request by ibkadeeko**
#### What does this PR do?
This pull request enables a user to comment on an article

#### Description of Task to be completed
- create comments container component
- create comments form component
- create single comment component

#### How should this be manually tested?
N/A

#### Any Background Context You want to provide?
N/A

#### What are the relevant pivotal tracker stories?

[#167377502](https://pivotaltracker.com/story/show/167377502)

#### Screenshots (if appropriate)
Screenshot 2019-09-08 at 13 41 24

#### Questions:
N/A

### CODE REVIEW

File: src/assets/scss/style.scss [hunk 1]
```diff
@@ -1,7 +1,13 @@
 @import '~sass-rem';
-@import url('https://fonts.googleapis.com/css?family=Rubik:400,700&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Rubik:300,400,500,700&display=swap');
 @import 'variables';
 @import 'mixins';
+@import '~semantic-ui-css/semantic.min.css';
+
+html {
+  font-family: 'Rubik', sans-serif;
+  font-size: 62.5%;
```
Comment (ibkadeeko) [inline src/assets/scss/style.scss]: remove this

File: src/assets/scss/style.scss [hunk 2]
```diff
@@ -15,3 +21,9 @@ body {
 .hover {
   color: $hovercolor;
 }
+
+@media screen and (max-width: 32rem) {
+  html {
+    font-size: 46.875%;
```
Comment (ibkadeeko) [inline src/assets/scss/style.scss:26]: remove this

File: src/components/Comments/Comment.jsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
```
Comment (ibkadeeko) [inline src/components/Comments/Comment.jsx]:
```suggestion
import React from 'react';
```

Comment (ibkadeeko) [reply src/components/Comments/Comment.jsx]: change in wiki also

File: src/components/Comments/Comments.spec.jsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    expect(commentForm.find('textarea')).toHaveLength(1);
+  });
+
+  it('should render properly without saving parameter', () => {
+    const commentForm = shallow(<CommentForm {...noSaving} />);
+
+    expect(commentForm.find('textarea')).toHaveLength(1);
+  });
+
+  it('should call handleSave function', () => {
```
Comment (ibkadeeko) [inline src/components/Comments/Comments.spec.jsx]:
```suggestion
  it('should call handleChange function', () => {
```

File: src/components/Comments/Comments.stories.jsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+
+import { storiesOf } from '@storybook/react';
+
+import { comments } from 'fixtures/articleComments';
+
+import { Comments } from 'components/Comments';
+
+storiesOf('Comments', module).add('default', () => (
```
Comment (ibkadeeko) [inline src/components/Comments/Comments.stories.jsx]: create for a single comment alone

File: src/store/modules/articleComments/index.js [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+export default (state = initialState, { type, payload }) => {
+  if (apiCallStarted(type)) {
+    return {
+      ...state,
+      apiCallsInProgress: state.apiCallsInProgress + 1
+    };
+  }
+  switch (type) {
+  case types.GET_COMMENTS_SUCCESS:
```
Comment (ibkadeeko) [inline src/store/modules/articleComments/index.js:76]: correct indentation

File: src/components/Spinner/Spinner.scss [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+  font-size: 1rem;
+  text-indent: -9999rem;
+  -webkit-transform: translateZ(0);
+  -ms-transform: translateZ(0);
+  transform: translateZ(0);
+  -webkit-animation: load8 1.1s infinite linear;
+  animation: load8 1.1s infinite linear;
+}
+
+@keyframes load8 {
```
Comment (ibkadeeko) [inline src/components/Spinner/Spinner.scss]: rename load8

File: src/store/modules/articleComments/index.js [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+
+export const createCommentSuccess = (comment) => ({
+  type: types.CREATE_COMMENT_SUCCESS, payload: { comment }
+});
+
+export const createCommentFailure = (error) => ({
+  type: types.CREATE_COMMENT_FAILURE, payload: { error }
+});
+
+export function* makeGetCommentsAPICall({ payload }) {
```
Comment (ibkadeeko) [inline src/store/modules/articleComments/index.js]:
```suggestion
export function* getComments({ payload }) {
```

File: src/store/modules/articleComments/index.js [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+  try {
+    const comments = yield call(API.getComments, payload);
+    yield put(getCommentsSuccess(comments));
+  } catch (error) {
+    yield put(getCommentsFailure(error));
+  }
+}
+
+export function* makeCreateCommentAPICall({ payload }) {
```
Comment (ibkadeeko) [inline src/store/modules/articleComments/index.js]: same as above

File: src/store/modules/articleComments/types.js [hunk 10]
```diff
@@ -0,0 +1,6 @@
+export const GET_COMMENTS_REQUEST = 'vali-ah-frontend/auth/GET_COMMENTS_REQUEST';
```
Comment (ibkadeeko) [inline src/store/modules/articleComments/types.js]:
```suggestion
export const GET_COMMENTS_REQUEST = 'vali-ah-frontend/articleComments/GET_COMMENTS_REQUEST';
```

Comment (ibkadeeko) [reply src/store/modules/articleComments/types.js]: change for others

File: src/store/modules/articleComments/index.js (hunk 6 above)
Comment (ibkadeeko) [reply src/store/modules/articleComments/index.js:76]: This is the indentation that passes eslint for this

**Review by Bastieno — APPROVED** [review]
This is so outdated. Close this if you can.

**Pull request closed without merging**